ui/src/containers/WorkflowParametersDialog.css

```diff
@@ -5,3 +5,76 @@
 .rjsf #root > .form-group {
   margin-bottom: 15px !important;
 }
+
+.title_box {
+  border: #3c5a86 1.5px dotted;
+}
+
+/* .gphl_row_box > .crystal_data,
+.gphl_row_box > .parameters{ */
+.gphl_row_box > .indexing_solution {
+  border: #e74444 1.5px dotted !important;
+}
+
+.title_box #title {
+  position: relative;
+  color: #01012e;
+  font-size: large;
+  top: -0.7em;
+  margin-left: 1em;
+  display: inline;
+  background-color: white;
+}
+
+.indexing_table {
+  width: 100%;
+}
+
+.indexing_table > thead,
+.indexing_table > tbody,
+.indexing_table > tr,
+.indexing_table > td,
+.indexing_table > th {
+  display: block;
+}
```

Comment:
Any way you could work _with_ the CSS table layout instead of against it? If the problem is to scroll the body and freeze the header row, you could make the table's container scrollable and use `position: sticky` on the header row. Just a thought, I don't have strong feeling about it :grin: